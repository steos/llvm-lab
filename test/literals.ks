	
	void main()
	{
		int hex = 0x0a;
		int hex2 = 0xf7;
		int oct = 017;
		int oct2 = 0713;
		iprintln(hex);
		iprintln(hex2);
		iprintln(oct);
		iprintln(oct2);
	}
	
	native void iprintln(int);