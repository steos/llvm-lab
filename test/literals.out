10
247
15
459
